app/models/payment.rb

```diff
@@ -4,6 +4,7 @@ class Payment < ApplicationRecord
   attribute :amount, :pence
   attribute :payment_type
   attribute :frequency, :value_object, source: PaymentFrequencyType
+  attribute :ownership_type
```

Comment:
do i need to add `:value_object, source: OwnershipType?` here

Reply:
If you want it casting correctly you do, but it does not make that much difference unless your asking .partner? question etc.